randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,12 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          Morph Ball
+          Any of the following:
+              Grapple Beam and Spider Magnet
+              All of the following:
+                  Flash Shift
+                  Grapple Beam or Spider Magnet or Walljump (Beginner)
```

Comment:
Beginner walljumps like this are treated as being trivial now, so this can just be flash shift. You can also walljump after using grapple. Space Jump is a viable option as well. So it should be Grapple or Flash Shift or Space Jump.

Reply:
I've unsuccessfully tried to pull it off with just space jump before, but after you said it here I went back and was able to do it without much trouble.

I also figured out that you can get this by using the same cross bomb jump that can be used to pass through shutters (flash shift doors), to launch you into the spider surface. And you can charge speed in the save room. I can't immediately tell if wall jumping off from either of those actions is viable, so at least there's some usefulness to the spider surface.